<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackControllerHW_TestBench</class>
 <widget class="QMainWindow" name="TrackControllerHW_TestBench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Track Controller (HW) - TestBench</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="labelTestInputs">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Test Inputs</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelSpeedInput">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Suggested Speed (mph):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSpeedInput">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelAuthorityInput">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Received Authority (ft):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAuthorityInput">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelOccupiedInput">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Occupied Block (ID):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxOccIn">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelFailedInput">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Failed Block (ID):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxFailedIn">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="labelTestOutputs">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Test Outputs</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelCommandedOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Commanded Speed (mph):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSpeedOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="labelAuthorityOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Commanded Authority (ft):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAuthOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="labelOccupiedOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Occupied Blocks:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOccupiedOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearOcc">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="labelFailedOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Failed Blocks:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFailedOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearFailed">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelCheckBlock">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Check Block States:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxCheckBlock">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="labelCrossingOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Crossing:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCrossingOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="labelSwitchOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Switch:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSwitchOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="labelLightOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Light:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditLightOut">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
